src/backend/joanie/core/serializers/admin.py

```diff
@@ -163,7 +163,9 @@ class AdminProductSerializer(serializers.ModelSerializer):
         coerce_to_string=False, decimal_places=2, max_digits=9, min_value=0
     )
     price_currency = serializers.SerializerMethodField(read_only=True)
-    instructions = serializers.CharField(required=False)
+    instructions = serializers.CharField(
+        required=False, allow_blank=True, trim_whitespace=False
```

Comment:
Shouldn't we either allow empty strings or stop requiring a not None value?
I you do both then we can have None or "" and that's not what we want?
I'm not sure what we is the expected behavior but maybe we should add tests on both cases after we decide what we want.